### PR TITLE
feat(dh): refresh users in Admin when user profile is updated

### PR DESCRIPTION
**Pull request by dzhavat**
### DataHub
- refresh users in Admin when user profile is updated
- update translation

## References

- N/A

### CODE REVIEW

File: libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts [hunk 1]
```diff
@@ -136,6 +138,10 @@ export class DhUsersTabComponent {
     this.userRolesStore.getRoles();
 
     this.onSearchInput();
+
+    this.profileModalService.onProfileUpdate$
+      .pipe(takeUntilDestroyed(this.destroyRef))
```
Comment (mimse) [inline libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts:143]: Suggestion: No need for this.destoryRef in constructor

File: libs/dh/profile/feature-profile-modal/src/lib/dh-profile-modal/dh-profile-modal.service.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DhProfileModalService {
+  private profileUpdateSubject = new Subject<void>();
+
+  public onProfileUpdate$ = this.profileUpdateSubject.asObservable();
```
Comment (mimse) [inline libs/dh/profile/feature-profile-modal/src/lib/dh-profile-modal/dh-profile-modal.service.ts:26]: Question: Isnt a subject already a observable?

**Review by mimse — CHANGES_REQUESTED** [review]

File: libs/dh/profile/feature-profile-modal/src/lib/dh-profile-modal/dh-profile-modal.service.ts (hunk 2 above)
Comment (dzhavat) [reply libs/dh/profile/feature-profile-modal/src/lib/dh-profile-modal/dh-profile-modal.service.ts:26]: Yes, it is. The `asObservable()` pattern is for protecting the subject, so its `next` is only called via the method below.

File: libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts (hunk 1 above)
Comment (dzhavat) [reply libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts:143]: Nice catch. Should be fixed now.

**Review by mimse — APPROVED** [review]

**Pull request merged**